src/WhatToCook.Application/DataTransferObjects/Responses/ShoppingListResponse.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace WhatToCook.Application.DataTransferObjects.Responses
+{
+    public class ShoppingListResponse
+    {
+        public IEnumerable<string> Ingredients { get; set;}
```

Comment:
Could be initialized with some default value, good approach would be to use: https://learn.microsoft.com/pl-pl/dotnet/api/system.linq.enumerable.empty?view=net-7.0

Reply:
Seems to be fixed